### PR TITLE
still need to update virtualenv with python3 instructions

### PR DESCRIPTION
**Pull request by glassresistor**
Someone check for typos and readability.

### CODE REVIEW

File: Python.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+PIP and Environment Management
+---
+Use ``pip`` to install applications and [virtualenv](https://github.com/codeforamerica/howto/blob/master/Python-Virtualenv.md) to manage your development environments.
+
+Unless otherwise required use Python 3.5 and if multiple version of support are desired use [tox](https://tox.readthedocs.org/en/latest/) to run tests across multiple ``python`` versions.
+
+Project Structure
+-----------------
+All projects should include README.md files with links to this document, Contributors.md with information about how to contribute and who is contributing, and a license file, MIT preferred.  Each project should have a requirements folder containing a app.txt file in the [requirement file format](https://pip.readthedocs.org/en/1.1/requirements.html) always pin requirements to and use projects which respect [semvar](http://semver.org/).  Also including a dev.txt requirements file with developer tools can be helpful as well as operating system specific scripts.
```
Comment (migurski) [inline Python.md]: `s/semvar/semver/`

File: Python.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+Python
+======
+Python is a clear and powerful object-oriented programming language, comparable to Perl, Ruby, Scheme, or Java.  Its main focus is on readability and standards based approached
+
+PIP and Environment Management
+---
+Use ``pip`` to install applications and [virtualenv](https://github.com/codeforamerica/howto/blob/master/Python-Virtualenv.md) to manage your development environments.
```
Comment (migurski) [inline Python.md]: Relative `Python-Virtualenv.md` is probably okay for the link, in case this gets forked someplace sometime.

Comment (glassresistor) [inline Python.md]: thanks mike doing it now

**Comment by glassresistor** [comment]
@bsmithgall  tagged?

**Comment by bsmithgall** [comment]
Hi :wave:

**Comment by bsmithgall** [comment]
Ack -- really sorry I didn't get a chance to take a look at this :(.

**Pull request merged**